recipes/themysto/meta.yaml

```diff
@@ -0,0 +1,83 @@
+# this is a conda-forge recipe. See https://github.com/conda-forge for details
+
+{% set version = "0.0.4" %}
+
+package:
+  name: themysto
+  version: {{ version }}
+
+source:
+  # also taken from nb_conda
+  git_url: https://github.com/jcb91/IPython-notebook-extensions.git
+  git_tag: conda
+
+build:
+  number: 0
+  # we can't use noarch_python because it places prohibitive restrictions:
+  #  - can't specify supported version
+  #  - attempts to roll its own pre/post-link scripts, complaining about existing ones.
+  entry_points:
+    # Don't put entry points here, as conda bungles the script creation.
+    # Instead rely on setuptools.
+    # The syntax is module:function
+    # - themysto = themysto.application:main
+
+requirements:
+  build:
+    - python 2.7*|3.3*|3.4*|3.5*
+    - ipython_genutils
+    - jupyter_core
+    - nbconvert
+    - notebook >=4.0
+    - psutil >=2.2.1
+    - pyyaml
+    - setuptools
+    - tornado
+    - traitlets
+
+  run:
+    - python 2.7*|3.3*|3.4*|3.5*
+    - ipython_genutils
+    - jupyter_core
+    - nbconvert
+    - notebook >=4.0
+    - psutil >=2.2.1
+    - pyyaml
+    - setuptools
+    - tornado
+    - traitlets
+
+test:
+
+  # Python imports
+  imports:
+    - themysto
+    - themysto.extensions
+    - themysto.nbextensions_configurator
+    - themysto.postprocessors
+    - themysto.preprocessors
+
+  commands:
+    # test commands to be run.  Used to test that the entry points work.
+    - themysto --help
+    - themysto install --help
+    - themysto uninstall --help
+    - themysto retire --help
+
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time. We don't, because conda doesn't provide tox or selenium, so
+  # it's pretty useless for running tests. Rely instead on CI testing.
+
+about:
+  summary: "jcb91's fork of ipython-contrib/IPython-notebook-extensions"
+  home: https://github.com/jcb91/IPython-notebook-extensions.git
```

Comment:
Could we please use the upstream version ( https://github.com/ipython-contrib/IPython-notebook-extensions )? Also, `.git` should be dropped.